### PR TITLE
Expanded the oc completion page to include the setup steps for both B…

### PR DESCRIPTION
**Pull request by bergerhoffer**
…ash, and Zsh shell's

Adapted from #47715.

Version(s):
4.6+

Issue:
#47714

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/pr-47715/cli_reference/openshift_cli/configuring-cli.html

### CODE REVIEW

File: modules/cli-configuring-completion-zsh.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+After you install the OpenShift CLI (`oc`), you can enable tab completion to automatically complete `oc` commands or suggest options when you press Tab. The following procedure enables tab completion for the Zsh shell.
+
+.Prerequisites
+
+* You must have the OpenShift CLI (`oc`) installed.
+
+.Procedure
+
+* Run the following command to add tab completion for `oc` to your `.zshrc` file.
```
Comment (jeana-redhat) [inline modules/cli-configuring-completion-zsh.adoc:17]:
```suggestion
* To add tab completion for `oc` to your `.zshrc` file, run the following command:
```

**Review by jeana-redhat — COMMENTED** [review]
Just one small suggestion, LGTM

File: modules/cli-configuring-completion-zsh.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/cli-configuring-completion-zsh.adoc:17]: Thanks!

**Comment by bergerhoffer** [comment]
@zhouying7780 Can you please QE review this update for zsh tab completion?

Preview: http://file.rdu.redhat.com/~ahoffer/2022/pr-47715/cli_reference/openshift_cli/configuring-cli.html

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Pull request merged**